debian_package/sbin/folder2ram

```diff
@@ -234,67 +235,40 @@ exit
 EOF
 
   #chmodding this script to make it only root/group-writable and root/group-executable
-  chmod 774 "/etc/init.d/folder2ram_temporary"
+  chmod 755 "/etc/init.d/folder2ram_temporary"
 
 } #### END write_cleanup_initscript
 
 write_systemd_startup_service() {
 
-  if [ "$timeout_setting" != '' ]; then
-    timeout_line="TimeoutSec=$timeout_setting"
-  fi
+  #read timeout from configuration file
+  local timeOUT=$(read_config_file | grep "^TIMEOUT" | awk -F'=' '{print $2}')
+  [ "${timeOUT}" != "" ] || timeOUT=120
 
   #writing a tiny amount of stuff to have systemd do the same thing as above.
   #clear symptom that systemd is master race.
   cat <<EOF >"$systemd_startup_service_file"
 [Unit]
 Description=folder2ram systemd service
 After=local-fs.target
-After=blk-availability.service
-DefaultDependencies=no
```

Comment:
This is also part of the settings needed to split the systemd unit in two so it is run at the right time. See the comment where you deleted the systemd folder2ram shutdown unit for more details.

please don't delete this line